plugins-compat-tester-model/src/main/java/org/jenkins/tools/test/model/PluginCompatTesterConfig.java

```diff
@@ -317,11 +316,16 @@ public void setMavenPropertiesFiles( String mavenPropertiesFile ) {
         return res;
     }
 
+    private String getTestJavaCommandPath() {
+        return new File(testJDKHome, "bin/java").getAbsolutePath();
```

Comment:
may explode if the parameter is null

Reply:
Fixed, thanks for the heads-up @oleg-nenashev